docs/faq.md

```diff
@@ -91,11 +100,11 @@ For now ExternalDNS uses TXT records to label owned records, and there might be
 
 ### Does anyone use ExternalDNS in production?
 
-Yes — Zalando replaced [Mate](https://github.com/linki/mate) with ExternalDNS since its v0.3 release, which now runs in production-level clusters. We are planning to document a step-by-step tutorial on how the switch from Mate to ExternalDNS has occurred.
+Yes — Zalando replaced [Mate](https://github.com/linki/mate) with ExternalDNS since its v0.3 release, which now runs in production-level clusters.
```

Comment:
I know of a bunch of other companies using External DNS in production, e.g. @stealthybox uses it with GKE --- maybe we can add one sentence mentioning multiple production users (at least known for AWS and GKE)

Reply:
I thought about that as well. Shall we make a list or just get rid of that? I think we wrote it at the very beginning, but right now we don't really need that at all IMO.